api-server/src/server/middlewares/csurf.js

```diff
@@ -1,21 +1,25 @@
 import csurf from 'csurf';
 
+export const csrfOptions = {
+  domain: process.env.COOKIE_DOMAIN || 'localhost',
+  sameSite: 'strict',
+  secure: process.env.FREECODECAMP_NODE_ENV === 'production'
```

Comment:
```suggestion
  secure: process.env.FREECODECAMP_NODE_ENV !== 'development'
```
This is slightly out of scope, but I think secure by default is a good principle.